API.md

```diff
@@ -123,6 +123,7 @@ Event object associated with 'error' events.
 - `method` - method of the request that originated the error. Maps to `request.method`.
 - `pid` - the current process id.
 - `error` - the raw error object.
+- `config` - plugin-specific config object combining `request.route.settings.plugins.good` and `request.plugins.good` using `Object.assign` with request-level configs overriding route-level configs. Can be used for additional filtering logic.
```

Comment:
Yeah you don't need to mention `Object.assign`. Just phrase it that request level overrides route level.